mjpc/planners/include.cc

```diff
@@ -26,6 +27,7 @@
 
 namespace mjpc {
 const char kPlannerNames[] =
```

Comment:
Move CEM to be the last planner. The planner indexing assumes the current ordering so putting CEM first will affect many of the example tasks.